src/indexing.c

```diff
@@ -255,6 +266,85 @@ indexing_create_and_verify_hypertable_indexes(Hypertable *ht, bool create_defaul
 	relation_close(tblrel, AccessShareLock);
 }
 
+extern ObjectAddress
+ts_indexing_root_table_create_index(IndexStmt *stmt, const char *queryString)
```

Comment:
Is this an exact copy of the postgres code? If so comment above function. If not, comment what's different

Reply:
comment was within the code block, moving above

Reply:
(not quite and exact copy, adapted for hypertables instead of declarative partitioning)